### PR TITLE
Search button styling

### PR DESCRIPTION
**Pull request by wachiya**

### CODE REVIEW

**Comment by wachiya** [comment]
home

A fix on the homepage search button styling.

File: src/dashboard/home-dashboard.component.tsx [hunk 1]
```diff
@@ -10,9 +10,9 @@ export default function HomeDashboard(props: HomeDashboardProps) {
         <UserHasAccessReact privilege="View Patients">
           <Link
             to={props.match.url + "/patient-search"}
-            className="omrs-link omrs-outlined-action omrs-rounded"
+            className={styles.link}
```
Comment (joeldenning) [inline src/dashboard/home-dashboard.component.tsx]: I would rather not write our own custom css when we have styleguide classes that already do this. Can you explain what these changes accomplish? Can any of the changes be moved to esm-styleguide?

Comment (wachiya) [reply src/dashboard/home-dashboard.component.tsx]: We were trying to expand the search button and center its content and yes most of the changes can be moved to the styleguide. I'll make the required changes right away.

Comment (joeldenning) [reply src/dashboard/home-dashboard.component.tsx]: 👍

Comment (wachiya) [reply src/dashboard/home-dashboard.component.tsx]: New fix up for review now.

File: src/dashboard/home-dashboard.component.tsx [hunk 2]
```diff
@@ -5,26 +5,26 @@ import { UserHasAccessReact } from "@openmrs/esm-api";
 
 export default function HomeDashboard(props: HomeDashboardProps) {
   return (
-    <div className={styles.homeDashboard}>
-      <section className={styles.mainSection}>
-        <UserHasAccessReact privilege="View Patients">
```
Comment (joeldenning) [inline src/dashboard/home-dashboard.component.tsx]: It looks like we accidentally removed UserHasAccessReact. Was that on purpose? If not, let's bring it back -- we need it.

Comment (wachiya) [reply src/dashboard/home-dashboard.component.tsx]: Not on purpose, let me get to it.

Comment (wachiya) [reply src/dashboard/home-dashboard.component.tsx]: Changes updated, please review.

**Pull request merged**